start.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```

Comment:
You need to `chmod +x` this file.

Reply:
This file is stored in git with executable permission and COPY preserves the file permission so we don't necessarily need it.